qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -358,6 +371,51 @@ def _to_sparse_label(label):
             filter(None, (label_transformation[c].format(i=i) for i, c in enumerate(label)))
         )
 
+    def to_normal_order(self) -> "FermionicOp":
+        """Convert to the equivalent operator with normal order.
+        The returned operator is a sparse label mode.
+
+        .. note::
+
+            This method does not return usual normal ordering.
+            The order is calculated by commutation relation.
+            For example, in usual :math:`\\colon c_0 c_0^\\dagger\\colon = -c_0^\\dagger c_0`
+            where :math:`c_0` is an annihilation operator,
+            but this method returns :math:`1 - c_0^\\dagger c_0`.
+            See the reference: https://en.wikipedia.org/wiki/Normal_order#Multiple_fermions.
```

Comment:
```suggestion
            This method implements the transformation of an operator to the normal ordered operator. 
            The transformation is calculated by considering all commutation relations between the operators.
            For example, for the case :math:`\\colon c_0 c_0^\\dagger\\colon`
            where :math:`c_0` is an annihilation operator,
            this method returns :math:`1 - c_0^\\dagger c_0` due to commutation relations. 
            See the reference: https://en.wikipedia.org/wiki/Normal_order#Multiple_fermions.
```

Reply:
@ikkoham thanks for the updates. I think the suggestion is more explanatory and helps understand better what we are doing. I still find the previous one a bit cryptic. @mrossinek what do you think?

Reply:
Thank you so much. I was only worried about confusing the users who know the wick ordering. I think your improvement is nice. After Max's approval, I'll update it.

Reply:
Thanks @pbark! I think your suggestion works very well :+1: 
@ikkoham if you can update this today, we can merge it right after :slightly_smiling_face: